File: app/Models/OrderTask.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class OrderTask extends Model
{
    protected $fillable = [
        'order_id',
        'title',
        'description',
        'status',
        'position',
    ];

    public function order()
    {
        return $this->belongsTo(Order::class);
    }
}

File: logikraf-v2/internal/domain/model/order_task.go
package model

import "time"

type OrderTask struct {
	ID          uint      `gorm:"primaryKey" json:"id"`
	OrderID     uint      `json:"order_id"`
	Title       string    `gorm:"size:255" json:"title"`
	Description string    `gorm:"type:text" json:"description"`
	Status      string    `gorm:"size:50;default:pending" json:"status"`
	Position    int       `gorm:"default:0" json:"position"`
	CreatedAt   time.Time `json:"created_at"`
	UpdatedAt   time.Time `json:"updated_at"`
	DeletedAt   *time.Time `gorm:"index" json:"-"`
}
